packages/daimo-common/src/foreignCoin.ts

```diff
@@ -34,6 +34,34 @@ export const USDbC: ForeignCoin = {
   logoURI: `https://daimo.com/assets/foreign-coin-logos/USDbC.png`, // CoinGecko logo is fugly
 };
 
+export const DAI: ForeignCoin = {
+  token: "0x50c5725949A6F0c72E6C4a641F24049A917DB0Cb",
+  fullName: "Dai Stablecoin",
+  symbol: "DAI",
+  decimals: 18,
+  logoURI:
+    "https://assets.coingecko.com/coins/images/9956/large/dai-multi-collateral-mcd.png?1574218774",
+};
+
+export const USDT: ForeignCoin = {
+  token: "0xfde4C96c8593536E31F229EA8f37b2ADa2699bb2",
+  fullName: "Tether USD",
+  symbol: "USDT",
+  decimals: 6,
+  logoURI:
+    "https://assets.coingecko.com/coins/images/325/large/tether.png?1547034089",
+};
+
+export const stablecoinCoins = new Map<string, ForeignCoin>([
+  [daimoUSDC.token, daimoUSDC],
+  [USDbC.token, USDbC],
```

Comment:
i'd delete USDbC

& maybe call this `supportedSendCoins` ? can receive any erc20, send only these listed ones

+ `supportedHomeCoins`